### PR TITLE
Fix/8921 color inconsistencies in dark mode

### PR DESCRIPTION
**Pull request by puneetmahali1**
## Description
There is some mismatch color on dark mode 6 screens. it's hard to see the difference most of the screen because it's seems like dark and light black color and we need more focused to see the color diffrence.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8921

## Screenshots
Screenshot 2022-10-10 at 15 28 17

### CODE REVIEW

**Comment by Ein-Tim** [comment]
@puneetmahali1 

Thanks for this PR Puneet!

Could you please take into consideration to also fix the cases reported in https://github.com/corona-warn-app/cwa-app-ios/issues/2690?

Maybe this PR even already does that?

**Comment by puneetmahali1** [comment]
> 

You're welcome @Ein-Tim. In https://github.com/corona-warn-app/cwa-app-ios/issues/2690: these are different screens which I am working on.

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift [hunk 1]
```diff
@@ -59,12 +59,16 @@ class AntigenTestProfileInputViewController: UITableViewController, FooterViewHa
 			guard let cell = tableView.dequeueReusableCell(withIdentifier: AntigenTestProfileInputDescriptionCell.reuseIdentifier, for: indexPath) as? AntigenTestProfileInputDescriptionCell else {
 				fatalError("Wrong cell")
 			}
+			
+			cell.backgroundColor = .enaColor(for: .background)
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift:63]: isn't it better to set the backgroundColor on cell level?

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift [hunk 2]
```diff
@@ -59,12 +59,16 @@ class AntigenTestProfileInputViewController: UITableViewController, FooterViewHa
...
 			return cell
 		case .inputFields:
 			guard let cell = tableView.dequeueReusableCell(withIdentifier: AntigenTestProfileInputCell.reuseIdentifier, for: indexPath) as? AntigenTestProfileInputCell else {
 				fatalError("Wrong cell")
 			}
 			
+			cell.backgroundColor = .enaColor(for: .background)
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift:70]: same

File: src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/DiaryAddAndEditEntryViewController.swift [hunk 3]
```diff
@@ -78,6 +78,8 @@ class DiaryAddAndEditEntryViewController: UITableViewController, UITextFieldDele
 		guard let row = Row(rawValue: indexPath.row) else {
 			fatalError("Something went horribly wrong")
 		}
+		cell.backgroundColor = .enaColor(for: .background)
+
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/DiaryAddAndEditEntryViewController.swift:82]:
```suggestion
```

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift (hunk 1 above)
Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift]: Thanks! changed it to content view.

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift (hunk 2 above)
Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift]: Thanks! changed it to content view.

File: src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/DiaryAddAndEditEntryViewController.swift (hunk 3 above)
Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/DiaryAddAndEditEntryViewController.swift:81]: This is also in content view now.

**Review by naveeddotio — APPROVED** [review]

**Review by flxschmidt — APPROVED** [review]

**Pull request merged**